.github/workflows/ci.yml

```
@@ -25,12 +39,16 @@ jobs:
       - uses: actions/setup-node@v6
         with:
           node-version: "22.23.1"
+      - name: Configure isolated PostgreSQL integration URL
+        run: printf '%s=%s%s\n' FLOWOPS_TEST_DATABASE_URL postgres '://flowops_ci@127.0.0.1:5432/flowops_ci?sslmode=disable' >> "$GITHUB_ENV"
       - name: Verify formatting
         run: test -z "$(gofmt -l $(git ls-files '*.go'))" || (gofmt -l $(git ls-files '*.go') && exit 1)
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2046:warning:1:21: Quote this to prevent word splitting

(shellcheck)